tests/integration/optimizers/optimizer_conf.yml

```diff
@@ -0,0 +1,14 @@
+!OptunaOptimizer
+version: 1
+with:
+  flow_runner: !SingleFlowRunner
+    with:
+      flow_yaml: 'tests/integration/optimizers/flow.yml'
+      overwrite_workspace: True
+      documents: 'tests/integration/optimizers/data.jsonlines'
+      request_size: 1
+      task: 'search_lines'
+  evaluation_callback: !MeanEvaluationCallback {}
+  parameter_yaml: 'tests/integration/optimizers/parameter.yml'
+  workspace_base_dir: './optimizer_workspace'
```

Comment:
this should be parametrized to a tmpdir right?

Reply:
was just a draft